Gruntfile.js

```diff
@@ -605,9 +626,10 @@ module.exports = function(grunt) {
     ]
   );
 
-  grunt.registerTask('jade_compile', ['jade', 'notify:jade_compile']);
+  grunt.registerTask('pug_compile', ['pug', 'notify:pug_compile']);
```

Comment:
Added in https://github.com/Dogfalo/materialize/pull/4924/commits/83a44ce74bef1c450732c4a89c1dbe3ddee57e02